modules/perception/production/conf/perception/camera/fusion_camera_detection_component.pb.txt

```diff
@@ -8,7 +8,7 @@ image_channel_num : 3
 enable_undistortion : false
 enable_visualization : false
 output_final_obstacles : true
-output_obstacles_channel_name : "/perception/obstacles"
+output_obstacles_channel_name : "/apollo/perception/obstacles"
```

Comment:
Don't change this default config now. The channel name need to be further considered.